.github/workflows/build-ffmpeg.yml

```diff
@@ -20,25 +20,28 @@ jobs:
       matrix:
         include:
           - os_type: linux
-            os: ubuntu-22.04
+            os: ubuntu-20.04 # Pinned for now until all platforms moved to 22.04
             arch: x86_64
             shell: bash
             cmake_generator: Unix Makefiles
             ffmpeg_extras: |
               --enable-amf \
+              --enable-cuda \
+              --enable-cuda_llvm \
               --enable-encoder=h264_amf,hevc_amf \
               --enable-encoder=h264_nvenc,hevc_nvenc \
               --enable-encoder=h264_vaapi,hevc_vaapi \
-              --enable-encoder=libx264,libx265 \
+              --enable-ffnvcodec \
```

Comment:
Doesn't `nvenc` enable this automatically?

Reply:
No, without it I get `ERROR: nvenc requested, but not all dependencies are satisfied: ffnvcodec`

Reply:
In your local environment or in github? I think that'a path issue, and solved already by setting the path here: https://github.com/LizardByte/build-deps/blob/master/.github/workflows/build-ffmpeg.yml#L201

and using it here: https://github.com/LizardByte/build-deps/blob/f0b8147b5eebd1753fb20c52f32163093d5c8e2a/.github/workflows/build-ffmpeg.yml#L210

Reply:
Github CI, here's the log after I made the disable-autodetect change: https://github.com/brad-richardson/sunshine-build-deps/actions/runs/3550651639/jobs/5964224695